modules/monix/src/main/scala/retry/Monix.scala

```diff
@@ -1,16 +1,12 @@
 package retry
 
 import monix.eval.Task
-import monix.execution.{CancelableFuture, Scheduler}
 
 import scala.concurrent.duration.FiniteDuration
 
 object Monix {
 
-  implicit def sleepUsingScheduler(
-      implicit
-      scheduler: Scheduler
-  ): Sleep[CancelableFuture] =
-    (delay: FiniteDuration) => Task.sleep(delay).runAsync
+  implicit def sleepUsingScheduler: Sleep[Task] =
```

Comment:
This no longer needs to be a `def`, and could be renamed to `taskSleep` or something